pretraining_and_finetuning/transforms.py

```diff
@@ -0,0 +1,142 @@
+            # this is nnUNet's BrightnessMultiplicativeTransform
+            transforms.RandFlipd(keys=all_keys, prob=0.3, ),
+        ]
+    elif task == "finetuning":
+
+        # NOTE: the fine-tuning is done for lesion segmentation
+        all_keys = ["image", "label_sc", "label_lesion"]
+
+        monai_transforms = [
+            # pre-processing
+            transforms.LoadImaged(keys=all_keys),
+            transforms.EnsureChannelFirstd(keys=all_keys),
+            transforms.Orientationd(keys=all_keys, axcodes="RPI"),
+            transforms.CropForegroundd(keys=all_keys, source_key="label_sc"),   # crop everything outside the SC mask
+            # NOTE: spine interpolation with order=2 is spline, order=1 is linear
+            transforms.Spacingd(keys=all_keys, pixdim=(0.8, 0.8, 0.8), mode=(2, 1, 1)),
+            transforms.ResizeWithPadOrCropd(keys=all_keys, spatial_size=crop_size),
+            transforms.DivisiblePadd(keys=all_keys, k=2**5),   # pad inputs to ensure divisibility by no. of layers nnUNet has (5)
+            transforms.NormalizeIntensityd(keys=["image"], nonzero=False, channel_wise=False),
+            transforms.EnsureTyped(keys=all_keys, track_meta=False),
+            # sample patches from cropped image
+            transforms.RandCropByPosNegLabeld(keys=all_keys,
+                                              label_key="label_sc",  # cropping around the SC
+                                              spatial_size=patch_size,
+                                              pos=1, neg=0,
+                                              # if num_samples=4, then 4 samples/image are randomly generated
+                                              num_samples=4,
+                                              image_key="image", image_threshold=0),
+            # data-augmentation
+            # transforms.RandAffined(keys=all_keys, mode=(2, 1), prob=0.9,
+            #             rotate_range=(-20. / 360 * 2. * np.pi, 20. / 360 * 2. * np.pi),    # monai expects in radians 
+            #             scale_range=(-0.2, 0.2),   
+            #             translate_range=(-0.1, 0.1)),
+            # transforms.Rand3DElasticd(keys=all_keys, prob=0.5, sigma_range=(3.5, 5.5), magnitude_range=(25., 35.)),
+            transforms.RandSimulateLowResolutiond(keys=["image"], zoom_range=(0.5, 1.0), prob=0.25),
+            transforms.RandAdjustContrastd(keys=["image"], gamma=(0.5, 3.), prob=0.25),  # this is monai's RandomGamma
+            # transforms.RandBiasFieldd(keys=["image"], coeff_range=(0.0, 0.5), degree=3, prob=0.3),
+            # transforms.RandGaussianNoised(keys=["image"], mean=0.0, std=0.1, prob=0.1),
+            transforms.RandGaussianSmoothd(keys=["image"], sigma_x=(0., 2.), sigma_y=(0., 2.), sigma_z=(0., 2.0),
+                                           prob=0.3),
+            transforms.RandScaleIntensityd(keys=["image"], factors=(-0.25, 1), prob=0.15),
```

Comment:
Just to double-check; these four transformations are done only on `image`, right?